ForumProject/notifications/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+from notifications.models import Notification
+
+@admin.register(Notification)
+class NotificationAdmin(admin.ModelAdmin):
+    list_display = ['id', 'date_time', 'project', 'startup_name', 'investor', 'trigger', 'initiator']
```

Comment:
Consider the admin interface for this model should be user-friendly and displays relevant information, potentially leveraging `search_fields`, and list_filter to improve the usability of the admin for `Notification` model